![imagen](./img/ejercicios.png)

# Ejercicios probabilidad

## Ejercicio 1
#### Eventos mutuamente excluyentes
En este ejercicio vamos a calcular las probabilidades conjuntas que tienen de ganar la liga de fútbol inglesa tres equipos. Suponiendo que las posibilidades de ganar el torneo para cada equipo serían:

* Liverpool: P(A) = 1/3
* Manchester City: P(B) = 1/5
* Chelsea: P(C) = 1/9

Se pide:

1. Calcula la probabilidad de que gane la liga el Liverpool o el Manchester City
2. Calcula la probabilidad de que gane la liga el Liverpool, el Manchester City o el Chelsea
3. ¿Cuál es la probabilidad de que ninguno de estos equipos gane la liga?
4. ¿Cuál es la probabilidad de que no ganen ni Liverpool ni Mancherster City?

In [21]:
p1 = 1/3 + 1/5
print(p1)

p2 = 1/3 +1/5+1/9
print(p2)

p3 = 1- p2

p4 = 1 -p1

print(p3)
print(p4)

0.5333333333333333
0.6444444444444444
0.3555555555555556
0.4666666666666667


## Ejercicio 2
#### Eventos mutuamente excluyentes
Para este ejercicio se va a trabajar con una baraja francesa, con las siguientes características:

- En una baraja francesa hay diamantes, corazones, picas, y tréboles, siendo los diamantes y los corazones rojos, y las picas y los tréboles negros
- La baraja la forman 52 cartas
- La probabilidad de que salga un color u otro es de 26/52
- La probabilidad de que salga un palo u otro es de 13/52

Se pide:

1. ¿Cuál es la probabilidad de sacar una carta que sea corazón y de color negro?
2. ¿Cuál es la probabilidad de sacar una carta que sea corazón o de color negro?
3. ¿Cuál es la probabilidad de sacar una carta que sea corazón y que además sea un rey?
4. ¿Cuál es la probabilidad de sacar una carta que sea corazón o que además sea un rey?

In [23]:
c = 13/52
n = 26/52
k = 1/13

ej1= 0
ej2 = c+n
ej3 = c*k
ej4 = c+k-(c-k) #13/52 + 4/52
print(ej1)
print(ej2)
print(ej3)
print(ej4)

0
0.75
0.019230769230769232
0.15384615384615385


## Ejercicio 3
#### Eventos mutuamente NO excluyentes
Un vendedor de zumos decide terminar su jornada y regalar el resto de su mercancía a cada transeunte. Para esto sirve en 15 vasos todo el zumo que no se vendió y les coloca una tapa. Los deja en el mostrador para que cada persona tome el que prefiera.

Se sabe que el vendedor pudo llenar:

* 3 vasos con zumo de sandía (color rojo) { s1 , s2 , s3 }
* 6 vasos con naranja (color anaranjado) { n1 , n2 , n3 , n4 , n5 , n6 }
* 3 vasos con mango (color anaranjado) { m1 , m2 , m3 }
* 3 vasos con zumo de limón (color verde) { l1, l2 , l3 }

Definir la probabilidad de que al tomar un vaso ocurran los siguientes eventos:

1. Sea cítrico o anaranjado
2. Sea cítrico o verde
3. Sea de fruta o verde
4. No sea cítrico o sea anaranjado

In [28]:
p1 = 9/15 + 9/15 - (9/15 * 9/15)
p2 = 9/15 + 3/15 - (9/15 * 3/15) 
p3 = 15/15 + 3/15 - (1 *3/15)
p4 = (1-9/15) + 9/15 - (6/15 * 9/15)

print(p1)
print(p2)
print(p3)
print(p4)

citric_list = [ 'n1' , 'n2' , 'n3' , 'n4' , 'n5' , 'n6' ]
citric_list = [ 'n1' , 'n2' , 'n3' , 'n4' , 'n5' , 'n6' ]

0.84
0.68
1.0
0.76


## Ejercicio 4
#### Probabilidad condicional
Al 25% de tus amigos le gusta la fresa y el chocolate, mientras que al 60% le gusta el chocolate. ¿Cuál es la probabilidad de que a un amigo que le gusta el chocolate, le guste la fresa?

## Ejercicio 5
#### Probabilidad condicional
El 76 % de los estudiantes de Ingeniería Civil han aprobado resistencia de materiales y el 45 % aprobaron estática. Además, el 30 % aprobaron resistencia de materiales y estática. Si Camilo aprobó resistencia de materiales, ¿qué probabilidad tiene de haber aprobado también estática?

## Ejercicio 6
#### Probabilidad condicional
Si un hincha compra una camiseta del Manchester y se da cuenta de que está defectuosa, ¿cuál es la probabilidad de que sea del Manchester?

|Equipo/Camisetas|Buenas|Defectuosas|Total|
|------|-----|-----|----
|Juventus|508|92|600|
|Manchester|315|85|400|
|Total|823|177|1000|